tests/unit/bokeh/test_objects.py

```diff
@@ -270,7 +270,7 @@ def test_dataspec_field_in_json(self) -> None:
         json = obj.to_json(include_defaults=True)
         assert 'start_angle' in json
         assert 'start_angle_units' not in json
-        assert dict(units='rad', field='fieldname') == json['start_angle']
```

Comment:
Is this now omitted here because it is the default?

Reply:
Exactly, though still not ideal. I would like to replace (at some point due to backwards compatibility) all dicts that imitate structures with dataclasses (or similar), then it wouldn't matter how those tests are written, because `__eq__` would handle this.